examples/tests/multiple_tests.py

```diff
@@ -18,7 +18,7 @@ def test_hello(self):
         self.assertEqual(self.hello, "Hi there!")
 
     def testIdentity(self):
-        self.assertTrue(1, 1)
+        self.assertTrue(1 < 2)
```

Comment:
Given the name of this test, I believe the goal was something like `self.assertIs(1, 1)`.